api/Blob.json

```diff
@@ -98,10 +98,10 @@
               "version_added": "12"
             },
             "safari": {
-              "version_added": "8"
+              "version_added": "6"
```

Comment:
The existing data is paired Safari 5.1 / iOS 6. Should this be too, or is the existing data wrong?

Reply:
It appeared that Safari's data wasn't getting updated, but that's because our tests are returning `null` since neither `Blob` nor `BlobBuilder` are defined.  Updating all the data now!

Reply:
I think there was `WebKitBlob` originally, can you test some prefixed variants?

Reply:
I did some testing, no `WebKitBlob` in Safari 5.1, and also no `WebKitBlobBuilder` in Safari 6. Maybe that was never shipped even though it shows up in WebKit source history a lot.